pennylane/interfaces/batch/__init__.py

```diff
@@ -239,6 +240,10 @@ def execute(
             executed on a device. Expansion occurs when an operation or measurement is not
             supported, and results in a gate decomposition. If any operations in the decomposition
             remain unsupported by the device, another expansion occurs.
+        device_batch_transform (bool): Whether to apply any batch transforms defined by the device
+            (within :meth:`Device.batch_transform`) to each tape to be executed. The default behaviour
+            of the device batch transform is to expand out Hamiltonian measurements into
+            constituent terms if not supported on the device.

```

Comment:
Even though I can't think of any current use-cases, I figured it might be useful to be able to turn this new behaviour off (e.g., if bugs arise after a release, we have the ability to turn this off and on)